konig-schemagen/src/main/java/io/konig/schemagen/sql/SqlTableGenerator.java

```diff
@@ -37,19 +51,19 @@
 	
 	private static final Logger LOG = LoggerFactory.getLogger(SqlTableGenerator.class);
 	private SqlTableNameFactory nameFactory;
-	private SqlDatatypeMapper datatypeMapper;
+	private SqlDatatypeMapper datatypeMapper;
+	private Map<String,String> abbreviations;
 	
 	public SqlTableGenerator() {
 		nameFactory = new SqlTableNameFactory();
 		datatypeMapper = new SqlDatatypeMapper();
-	}
-	
+	}	
 	public SqlTableGenerator(SqlDatatypeMapper datatypeMapper) {
 		nameFactory = new SqlTableNameFactory();
 		this.datatypeMapper = datatypeMapper;
 	}
 	
-	public SqlTable generateTable(Shape shape) throws SchemaGeneratorException {
+	public SqlTable generateTable(Shape shape,File abbrevDir) throws SchemaGeneratorException {

```

Comment:
Don't pass the abbrevDir and load abbreviations locally.
Instead, create an AbbreviationsManager object and pass that.
The AbbreviationsManager should receive a Graph in its constructor, and then lazy load the abbreviations into a Map for a given ConceptScheme on demand.